wallet/src/main/java/com/android/identity_credential/wallet/credman/CredmanPresentationActivity.kt

```diff
@@ -400,14 +405,18 @@ class CredmanPresentationActivity : FragmentActivity() {
         mdocCredential: MdocCredential,
         consentFields: List<ConsentField>,
         trustPoint: TrustPoint?,
+        websiteOrigin: String?,
```

Comment:
Needs an entry in the function docstring.

Reply:
Ditto for `trustPoint`. Done.